cpp/pagerank_module/algorithm_online/pagerank.cpp

```diff
@@ -265,42 +308,40 @@ std::vector<std::pair<std::uint64_t, double>> SetPagerank(const mg_graph::GraphV
   return CalculatePageRank();
 }
 
-std::vector<std::pair<std::uint64_t, double>> GetPagerank(const mg_graph::GraphView<> &graph) {
+std::vector<std::pair<std::uint64_t, double>> GetPageRank(const mg_graph::GraphView<> &graph) {
   if (pagerank_online_alg::context.IsEmpty()) {
-    return SetPagerank(graph);
+    return SetPageRank(graph);
   }
-  if (IsIncosistent(graph)) {
+  if (IsInconsistent(graph)) {
     throw std::runtime_error(
-        "Graph has been modified, therefore is incosistent with cached results, please update the Pagerank by calling "
+        "Graph has been modified and is thus inconsistent with cached PageRank scores. To update them, please call "
         "set/reset!");
   }
   return CalculatePageRank();
 }
 
-std::vector<std::pair<std::uint64_t, double>> UpdatePagerank(
-    const mg_graph::GraphView<> &graph, const std::vector<std::uint64_t> &new_vertices,
-    const std::vector<std::pair<std::uint64_t, uint64_t>> &new_edges,
-    const std::vector<std::uint64_t> &deleted_vertices,
-    const std::vector<std::pair<std::uint64_t, uint64_t>> &deleted_edges) {
-  if (pagerank_online_alg::context.IsEmpty()) {
-    return SetPagerank(graph);
-  }
-
-  for (const auto &edge : deleted_edges) {
-    UpdateDelete(graph, edge);
+std::vector<std::pair<std::uint64_t, double>> UpdatePageRank(
+    const mgp::Graph &graph, const std::vector<std::uint64_t> &new_nodes,
+    const std::vector<std::pair<std::uint64_t, uint64_t>> &new_relationships,
+    const std::vector<std::uint64_t> &deleted_nodes,
+    const std::vector<std::pair<std::uint64_t, uint64_t>> &deleted_relationships) {
+  for (const auto &relationship : deleted_relationships) {
+    UpdateDelete(graph, relationship);
   }
-  for (const auto vertex : deleted_vertices) {
-    UpdateDelete(graph, vertex);
+  for (const auto node : deleted_nodes) {
```

Comment:
the reference here same as above?

```
 for (const auto &relationship : deleted_relationships) {
    UpdateDelete(graph, relationship);
  }
```

Reply:
Node is a single uint64_t and not using references is thus more efficient